app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,9 @@
     <string name="title_vote">Vote</string>
     <string name="title_comment">Comment</string>
     <string name="title_description">Description:</string>
+    <string name="title_image">Image</string>
+    <string name="title_audio">Audio</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_audio` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
This will be of use in my next pull request.